software/fitting/ebv_independent_calc.py

```diff
@@ -0,0 +1,149 @@
+"""
+This script crawls through sims_sed_library and computes the unnormalized,
+un-extincted SDSS magnitudes and magNorms (magnitudes in the Imsim Bandpass)
+of every stellar spectrum (main sequence, white dwarf, and M/L/T dwarf).
+
+The results are output to the text file ebv_independent_data.txt, which is
+meant to be used by fit_bright_stars.cpp
+"""
+
+from __future__ import with_statement
+import numpy as np
+import os
+from lsst.utils import getPackageDir
+from lsst.sims.photUtils import Bandpass, Sed
+import time
+
+def get_kurucz_phys(sed_name):
+    """
+    Read in the name of a kurucz SED file.  Return it's
+    T_eff, metallicity, log(surface gravity)
+    """
+    if sed_name[1] == 'p':
+        metallicity_sgn = 1.0
+    elif sed_name[1] == 'm':
+        metallicity_sgn = -1.0
+    else:
+        raise RuntimeError('Cannot parse metallicity sign of %s' % sed_name)
+
+    new_name = sed_name.replace('.','_').split('_')
+
+    metallicity = 0.1*metallicity_sgn*float(new_name[0][2:])
+
+    teff = float(new_name[-2])
+
+    logg = 0.1*np.float(new_name[3][1:])
+
+    return teff, metallicity, logg
+
+
+def get_wd_phys(sed_name):
+    """
+    Read in the name of a white dwarf SED,
+    return its T_eff, metallicity (which we don't actually have),
+    and log(surface gravity)
+    """
+    new_name = sed_name.replace('.','_').split('_')
+    teff = float(new_name[-2])
+    if new_name[1]!='He':
+        logg = 0.1*float(new_name[2])
+    else:
+        logg = 0.1*float(new_name[3])
+
+    return teff, -999.0, logg
```

Comment:
The documentation said a NULL value was -99, so you might want to use that here too.